R/module_teal.R

```diff
@@ -136,6 +138,27 @@ srv_teal <- function(id, modules, teal_data_rv, filter = teal_slices()) {
       title = "SessionInfo"
     )
 
+    if (!(is_mocked(session) || is_in_test())) {
+      output$lockFile <- downloadHandler(
+        filename = function() {
+          "renv.lock"
+        },
+        content = function(file) {
+          user_lockfile <- getOption("teal.renv.lockfile", "")
+          if (!file.exists(user_lockfile)) {
+            while (lockfile_task$status() == "running") {
+              Sys.sleep(0.25)
+            }
+            file.copy(lockfile_task$result(), file)
+            file
+          } else {
+            user_lockfile
+          }
+        },
```

Comment:
I think that the logs would be very useful here

Reply:
will try to rethink what we could put in here

Reply:
38406d14c8aadbae3d7e19687ef7fc9958a7e22d